crates/trycp_api/src/lib.rs

```diff
@@ -153,3 +159,22 @@ pub enum MessageToClient {
         response: std::result::Result<MessageResponse, String>,
     },
 }
+
+/// Messages returned directly by the TryCp server, rather than relayed from Holochain
+#[derive(Debug, serde::Deserialize, serde::Serialize)]
+#[serde(rename_all = "snake_case", tag = "type", content = "data")]
+pub enum TryCpServerResponse {
```

Comment:
Some type f*ry required to let the JS client decode these types. It's the first time we've returned anything more complicated than a string and the JS client has a special case for strings.